Nuro/Utilities/CoreData/Repository/ActivityLocalRepository.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ActivityLocalRepository.swift
+//  Nuro
+//
+//  Created by Karen Natalia on 12/10/22.
+//
+
+import Foundation
+import CoreData
+import UIKit
+
+protocol ActivityRepository {
+    func add(name: String, desc: String, image: UIImage, duration: Int, to category: Category)
+    func getActivitiesOfCategory(category: Category) -> [Activity]
+    func update(activity: Activity, newName: String, newDesc: String, newImage: UIImage, newDuration: Int, newCategory: Category)
+    func delete(activity: Activity)
+}
```

Comment:
Why not extract all CoreData protocols to another file? _Discussion_

Reply:
Yes, i agree 👍